Tests/NIOTests/RecvByteBufAllocatorTest.swift

```diff
@@ -136,8 +136,8 @@ final class AdaptiveRecvByteBufferAllocatorTest: XCTestCase {
         }
 
         let adaptive = AdaptiveRecvByteBufferAllocator(minimum: targetValue, initial: targetValue + 1, maximum: targetValue + 2)
-        XCTAssertEqual(adaptive.minimum, 1 << 31)
-        XCTAssertEqual(adaptive.maximum, 1 << 31)
-        XCTAssertEqual(adaptive.initial, 1 << 31)
+        XCTAssertEqual(adaptive.minimum, 1 << 30)
+        XCTAssertEqual(adaptive.maximum, 1 << 30)
+        XCTAssertEqual(adaptive.initial, 1 << 30)
```

Comment:
I had initially missed these tests that aren't run on armv7, got them after testing this pull on Android AArch64 too.